metrics/metrics.go

```diff
@@ -0,0 +1,136 @@
+package metrics
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"go.opentelemetry.io/otel/exporters/prometheus"
+	api "go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/sdk/metric"
+)
+
+const (
+	meterName     = "github.com/hyperledger-labs/yui-relayer"
+	namespaceRoot = "relayer"
+)
+
+var (
+	meterProvider *metric.MeterProvider
+	meter         api.Meter
+
+	ProcessedBlockHeightGauge      *Int64SyncGauge
```

Comment:
@siburu 
It would be helpful to write a succinct description for each metric here.

Reply:
oh, I noticed that they are written in the initialization of them.

Reply:
Yes but I should move to here! Thank you.